mono/mini/mini-runtime.c

```diff
@@ -3386,7 +3392,7 @@ MONO_SIG_HANDLER_FUNC (, mono_sigsegv_signal_handler)
 			return;
 
 		if (mono_dump_start ())
-			mono_handle_native_crash ("SIGSEGV", &mctx, (MONO_SIG_HANDLER_INFO_TYPE*)info);
+			mono_handle_native_crash (mono_get_signame (info->si_signo), &mctx, (MONO_SIG_HANDLER_INFO_TYPE*)info);
```

Comment:
also should be `signo`

Reply:
the `signo` won't work, it's out of scope here. I'll have to adjust this part a bit. Thought I could get away with `info->sisigno` since `info->si_si_addr` seems to work in this scope.